output/src/columns.rs

```diff
@@ -0,0 +1,35 @@
+use core::borrow::{Borrow, BorrowMut};
+use core::mem::{size_of, transmute};
+use valida_derive::AlignedBorrow;
+use valida_machine::Word;
+use valida_util::indices_arr;
+
+#[derive(AlignedBorrow, Default)]
+pub struct OutputCols<T> {
+    /// CPU clock
+    pub clk: T,
+
+    /// Output byte value
+    pub value: T,
+
+    /// Whether output operation is real write, not a dummy
+    pub is_real: T,
+
+    /// clk' - clk
+    pub diff: T,
```

Comment:
I think we can "inline" this since `clk' - clk` is degree 1

Reply:
I think we can't inline this at the moment, because the range check requires the difference to be in a single column. We should change that, but I'd vote to handle it in a separate PR.